TMS.Net07.Homework6.ShapeDraw/TMS.Net07.Homework6.ShapeDraw/Rectangle.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework6.ShapeDraw
+{
+    public class Rectangle : Shape
+    {
+        public Point A { get; }
+        public Point B { get; }
+        public Rectangle(Point a, Point b )
+        {
+            A = a ?? throw new ArgumentNullException(nameof(a));
+            B = b ?? throw new ArgumentNullException(nameof(b));
+            
+        }
+        public override double GetPerimeter()
+        {
+            var widht = GetWidht();
+            var hight = GetHight();
+            return (widht + hight) * 2;
+        }
+        public override double GetSquare()
+        {
+            var widht = GetWidht();
+            var hight = GetHight();
+            return widht * hight;
+        }
+        private int GetWidht()
+        {
+            return Math.Abs(A.X - B.X);
```

Comment:
We also could add two methods into `Point` class:
`GetHorizontalDistance()`
`GetVerticalDistance()`